Voice/AI Agent with NodeJS/index.js

```diff
@@ -0,0 +1,127 @@
+import * as dotenv from "dotenv";
+import fetch from "node-fetch";
+dotenv.config();
+import express from "express";
+import { RestClient } from "@signalwire/compatibility-api";
+const host = process.env.HOST_APP;
+
+const app = express();
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+app.post("/", (req, res) => {
+  const response = new RestClient.LaML.VoiceResponse();
+  const connect = response.connect();
+  const agent = connect.ai({
+    voice: "en-US-Neural2-D",
+  });
+  agent.setPostPromptURL("/summary");
+  agent.prompt(
+    {
+      confidence: 0.4,
+      frequencyPenalty: 0.3,
+    },
+    `You are the CEO's assistant.
+    Your job is to answer phone calls and collect messages or transfer the caller to a human agent. 
+    The CEO is not available now. Offer to take a message or transfer the caller to live agent.
+    Request the caller's name.
+    You are able to transfer a call to support or sales. 
+    Here is a list of known numbers that you can transfer the caller to. Do not ask the caller for these numbers. Just transfer the caller by using the appropriate function. 
+    | Name    | Phone Number    |
+    | --------| --------------- |
+    | Sales   | 12057937849     |
+    | Support | 19379028398     |
```

Comment:
In the guide, should we instruct the reader to go here and change the numbers?

Or as an alternative, could we add in the `.env` file a couple lines for `SALES_NUMBER` and `SUPPORT_NUMBER`, which users can configure?

Reply:
ahahah totally forgot to sanitize those. Thank you!